km/km_filesys.c

```diff
@@ -2201,7 +2298,11 @@ static inline int km_fs_recover_pipe(km_nt_file_t* nt_file, char* name)
       return -1;
    }
 
-   km_infox(KM_TRACE_SNAPSHOT, "recovered pipe: rfd=%d wfd=%d flags=0x%x", rfd, wfd, syscall_flags);
+   // Recover queued pipe contents
+   if (km_fs_recover_pipedata(nt_file, pipedata) != 0) {
```

Comment:
You simply pass the return on line 2276, an do this here. Any reason?

Reply:
Sloth!  Will fix.